sample/SampleRobot/samplerobot_auto_balancer.py

```diff
@@ -772,7 +772,7 @@ def demoStandingPosResetting():
 def demo():
     start_time = time.time()
     init()
-    from distutils.version import StrictVersion
+    from packaging.version import parse as StrictVersion
```

Comment:
What is the pupose of this chnage? to support `parse('"314.0.0"') > parse('"314.10.0"')` ? but it not work with `parse('"314.0.0"') > parse('314.10.0')`

Reply:
I know this problem when parsing a version string with double quatations. https://github.com/kirohy/hrpsys-base/commit/9488858a845967050ab0d34091348d478b2cffcc can avoid such situations.

This change is because you have already replace `distutils` into `packaging` in https://github.com/k-okada/hrpsys-base/commit/ecfefb92ab47866b35cf206e6064f9c06d338a2e#diff-9a55a4c37694757a5f9204a027881ffaa4793e2f5c144b586d7248ae5db441a5R15. Should I revert them? Then  https://github.com/kirohy/hrpsys-base/commit/10b4db8f66588b0a262b04791b41a6b53e9a696a will be deleted.

Reply:
If this works for both python2/python3 this is ok, if not, then we need to make changes as small as possibe and switch the code using python2/python3 conditional branches.